_sass/_layout.scss

```diff
@@ -255,7 +255,7 @@
 }
 
 .post {
-  padding: 40px 0 ;
+  padding: 80px 0 ;
```

Comment:
In the `.feature` selector (homepage header graphic area), we should also add `padding-top: 55px;` to match current spacing.